sources/procfs.go

```diff
@@ -675,6 +675,31 @@ func parseFileElements(path string, directoryDepth int) (name string, nodeName s
 	return name, nodeName, nil
 }
 
+func convertToBytes(s string) string {
+	new_s := ""
+	multiplier := 1
+	switch finalChar := s[len(s)-1:]; finalChar {
+	case "K":
+		new_s = strings.TrimSuffix(s, "K")
+		multiplier = 1000
+	case "M":
+		new_s = strings.TrimSuffix(s, "M")
+		multiplier = 1000000
+	case "G":
+		new_s = strings.TrimSuffix(s, "G")
+		multiplier = 1000000000
+	default:
+		//this should only happen for integers
+		return s
+	}
+	byteVal, err := strconv.Atoi(new_s)
+	if err != nil {
+		return s
+	}
+	byteVal = byteVal * multiplier
+	return strconv.Itoa(byteVal)
```

Comment:
Probably want to use the longer string/int parsing method we have used throughout this file as this breaks for sizes much greater than 1GB. For example, check out [this sample code](https://play.golang.org/p/Ro_bsdOFgn) for 10GB. It returns an unexpected number. Playing around with smaller values works as expected though.